posts/Jan/2022-04-29-mapping-basic-custom-type-in-hibernate-orm-600.adoc

```diff
@@ -0,0 +1,319 @@
+= Custom basic type mapping in Hibernate 6
+:icons: font
```

Comment:
There is additional metadata you need for a blog post -

```
:awestruct-tags: [ "Hibernate ORM" ]
:awestruct-layout: blog-post
```

I'd also maybe change the title.  Something like "Basic Vaue Mapping in ORM 6"